app/javascript/tests/enrollment/steps/Identification.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme';
+import Identification from '../../../components/enrollment/steps/Identification.js'
+import { mockPatient1 } from '../../mocks/mockPatients'
+
+const newEnrollmentState = {
+  isolation: false,
+  patient: { },
+  propagatedFields: {}
+}
+
+const requiredStrings = [
+  'WORKFLOW *',
+  'FIRST NAME *',
+  'MIDDLE NAME(S)',
+  'LAST NAME *',
+  'DATE OF BIRTH *',
+  'AGE',
+  'SEX AT BIRTH',
+  'GENDER IDENTITY',
+  'SEXUAL ORIENTATION',
+  'RACE (SELECT ALL THAT APPLY)',
+  'ETHNICITY',
+  'PRIMARY LANGUAGE',
+  'SECONDARY LANGUAGE',
+  'NATIONALITY',
+  'STATE/LOCAL ID',
+  'CDC ID',
+  'NNDSS LOC. REC. ID/CASE ID'
+]
+
+describe('Monitoree Identification', () => {
+  it('Properly renders all main components', () => {
+    const wrapper = mount(<Identification goto={() => {}} next={() => {}} setEnrollmentState={() => {}} currentState={newEnrollmentState} />);
+    requiredStrings.forEach(requiredString => {
+      expect(wrapper.text().includes(requiredString)).toBe(true);
+    })
+  });
+
+  it('Properly allows setting of all identification information', () => {
```

Comment:
Same feedback as above. If you want to check that all fields are set properly, you should test when all fields change. Also, to be more precise in these tests, I'd avoid just checking for these fields being falsy and instead check what they're actual expected values are.

Reply:
This was an intentional decision. Some of the fields start off as `""` while others are `null`. I  figured checking for their lack-of-existance was a good cover-all, consistent way to ensure they started off blank. I can go through and change them though.

Reply:
Yeah it's better that we know and expect all their initial values in case there are implications. The changes for app/javascript/tests/enrollment/steps/Contact.test.js look great, I think once this file is updated in the same way this should be good to go.

Reply:
I think this still needs to be addressed yeah? I see that the changing of the `middle_name`, `gender_identity,` and `sexual_orientation` aren't tested here.